server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -439,6 +449,30 @@ public Void withHandle(Handle handle)
     );
   }
 
+  @Override
+  public void removeTasksOlderThan(final long timestamp)
+  {
+    DateTime dateTime = DateTimes.utc(timestamp);
+    log.info("Deleting tasks older than [%s] and inactive at metastore.", dateTime.toString());
```

Comment:
Sorry, I was not clear. I think it's better to remove `log` from this class (which shouldn't be part of this PR) because it's not very useful. Logging should give us detailed context about what's happening, but a simple `I will delete some tasks from metadata` message is not much useful I think. Rather, it looks good to me to print logs in the caller of this method, like in `TaskAutoCleaner`.

Reply:
I agree. I think it's better to remove `log`.